app/controllers/movies_controller.rb

```diff
@@ -46,12 +46,12 @@ def show
 
   def modal
     @list = List.find(params[:list_id]) if params[:list_id].present?
-    @movie = Movie.find_by(tmdb_id: params[:tmdb_id]) || Tmdb::Client.movie(params[:tmdb_id])
+    @movie = GuaranteedMovie.find(params[:tmdb_id])
```

Comment:
I don't think we want to make this change, and this a great reason why we should rename stuff. 

`GuaranteedMovie.find` actually does `find or create` where as `Client.movie` either returns the persisted movie or an instance of `MovieMore`, which is not backed by a db table.

So in the case where the user searches and sees results (which may or may not be stored in the db), when clicking the modal, this would save the movie to the db rather than displaying data. 

That's not necessarily a bad change, but it's very difficult to know that behavior is changing just by looking at this diff.